test/integration/utils/buyCover.js

```diff
@@ -0,0 +1,27 @@
+const { getQuoteValues } = require('./getQuote');
+
+function coverToCoverDetailsArray (cover) {
+  return [cover.amount, cover.price, cover.priceNXM, cover.expireTime, cover.generationTime];
+}
+
+async function buyCover ({ cover, coverHolder, qt, p1 }) {
```

Comment:
why are parameters passed as a object instead of normal parameters?

Reply:
I usually use this pattern to leave room for new params. If any of those would be optional you wouldn't have to move all the optional params at the end and it also makes it clearer what each param is. In this case however if you think it's unlikely for it to change we can add them one by one.